frame/beefy/src/lib.rs

```diff
@@ -98,22 +98,27 @@ pub mod pallet {
 impl<T: Config> Pallet<T> {
 	/// Return the current active BEEFY validator set.
 	pub fn validator_set() -> ValidatorSet<T::BeefyId> {
-		ValidatorSet::<T::BeefyId> { validators: Self::authorities(), id: Self::validator_set_id() }
+		let validators: Vec<T::BeefyId> = Self::authorities();
+		let id: beefy_primitives::ValidatorSetId = Self::validator_set_id();
+		// Safe to unwrap() since `Self::authorities()` guaranteed to be non-empty
+		// by Self::initialize_authorities().
```

Comment:
The proof is incorrect, `initialize_authorities` will do nothing in case an empty validator set is being passed, but an empty vector will be a default storage value.